uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/BAO/DocumentTest.php

```diff
@@ -50,18 +50,6 @@ function testCreateDocumentWithNoSource() {
   /**
    * @expectedException CRM_Exception
    */
-  function testCreateDocumentWithNoAssignee() {
```

Comment:
Ah yes, silly of me. I added assertions - hopefully they work. I need to get testing set up on my machine soon